README.md

```diff
@@ -2,21 +2,21 @@
 
 A [Buildkite plugin](https://buildkite.com/docs/agent/v3/plugins) that lets you build, run and push build steps using [Docker Compose](https://docs.docker.com/compose/).
 
-* Containers are built, run and linked on demand using Docker Compose
-* Containers are namespaced to each build job, and cleaned up after use
-* Supports pre-building of images, allowing for fast parallel builds across distributed agents
-* Supports pushing tagged images to a repository
+-   Containers are built, run and linked on demand using Docker Compose
+-   Containers are namespaced to each build job, and cleaned up after use
+-   Supports pre-building of images, allowing for fast parallel builds across distributed agents
+-   Supports pushing tagged images to a repository
```

Comment:
This is our prettier, we might want to avoid this to reduce future merge conflicts.